content/en/cd-as-a-service/setup/quickstart.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Armory CD-as-a-Service Quickstart
+linktitle: Quickstart
+description: >
+  Install the Armory Continuous Deployment-as-a-Service CLI, connect your Kubernetes cluster with a single command, and deploy an example app using a traffic split. Learn deployment file syntax.
+weight: 1
+aliases:
+  - /cd-as-a-service/setup/get-started/
+  - /cd-as-a-service/setup/cli/
+---
+
+## Learning objectives
+
+1. [Sign up for CD-as-a-Service](#sign-up-for-cd-as-a-service).
+1. [Install the CD-as-as-Service CLI](#install-the-cd-as-as-service-cli) on your Mac or Linux workstation.
+1. [Connect your Kubernetes cluster](#connect-your-cluster) to CD-as-a-Service.
+1. [Deploy Armory's example app](#deploy-the-example-app) `potato-facts` to two environments: `staging` and `prod`.
+   * Use the CD-as-a-Service Console to approve an environment promotion.
+   * Observe a traffic split between two app versions.
+1. [Learn CD-as-a-Service deployment file syntax](#learn-deployment-file-syntax).
+1. [Remove](#clean-up) the resources you created. 
+
+## {{% heading "prereq" %}}
+
+* You are familiar with CD-as-a-Service's [key components]({{< ref "cd-as-a-service/concepts/architecture/key-components.md" >}}) and [system requirements]({{< ref "cd-as-a-service/concepts/architecture/system-requirements.md" >}}).
+* You have access to a Kubernetes cluster. If you need a cluster, consider installing a local [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/) or [Minikube](https://minikube.sigs.k8s.io/docs/start/) cluster.  Your cluster's API endpoint does not need to be publicly accessible to use CD-as-a-Service. 
```

Comment:
I'll also propose adding the text below, for users who are not ready to install kind, or are just exploring docs before sign up.
-  if you do not have a cluster and still wish to tour the product, You can sign up and select "Self-Guided Browser Tour". We spin a short-lived Kubernetes cluster to deploy the sample application.